plugins/modules/site_intent.py

```diff
@@ -934,75 +941,78 @@ def get_diff_deleted(self, config):
+        site_name = self.want.get("site_name")
 
         if site_exist:
             self.status = "success"
-            msg = "Requested Site - {0} present in Cisco DNA Center and creation verified.".format(self.want.get("site_name"))
-            self.log(msg)
+            self.msg = "Requested Site - {0} present in Cisco Catalyst Center and creation verified.".format(site_name)
+            self.log(self.msg, "INFO")
 
         require_update = self.site_requires_update()
 
         if not require_update:
-            self.log("Site - {0} Updation Verified Successfully.".format(self.want.get("site_name")))
+            self.log("Site - {0} Updation Verified Successfully.".format(site_name), "INFO")
             self. status = "success"
             return self
 
-        self.log("Playbook paramater doesnot match with the Cisco DNA Center means Merged task not executed successfully.")
+        self.log("Playbook paramater doesnot match with the Cisco Catalyst Center means Merged task not executed successfully.", "INFO")
 
         return self
 
     def verify_diff_deleted(self, config):
         """
-        Verify the deletion status of site configuration in Cisco DNA Center.
+        Verify the deletion status of site configuration in Cisco Catalyst Center.
         Args:
-            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
             - config (dict): The configuration details to be verified.
         Return:
-            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
         Description:
-            This method checks the deletion status of a configuration in Cisco DNA Center.
-            It validates whether the specified site exists in the DNA Center configuration.
+            This method checks the deletion status of a configuration in Cisco Catalyst Center.
+            It validates whether the specified site exists in the Catalyst Center configuration.
         """
 
         self.get_have(config)
-        self.log(str(self.have))
-        self.log(str(self.want))
+        self.log("Current State (have): {0}".format(str(self.have)), "INFO")
+        self.log("Desired State (want): {0}".format(str(self.want)), "INFO")
+
         # Code to validate dnac config for delete state
         site_exist = self.have.get("site_exists")
 
         if not site_exist:
             self.status = "success"
-            msg = "Requested Site - {0} already deleted from Cisco DNA Center and verified successfully.".format(self.want.get("site_name"))
-            self.log(msg)
+            msg = "Requested Site - {0} already deleted from Cisco Catalyst Center and verified successfully.".format(self.want.get("site_name"))
```

Comment:
msg = "The requested site {0} has already been deleted from the Cisco Catalyst Center and this has been successfully verified.".format(self.want.get("site_name"))

Reply:
added.